app/javascript/src/components/Dashboard/Contacts/Table/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+// @ts-ignore
+import { Table as NeetoUITable } from "neetoui";
+
+import { COLUMN_DATA, ROW_DATA } from "../constants";
+
+const Table = () => (
+  <NeetoUITable
+    className="text-gray-500"
+    columnData={COLUMN_DATA}
+    currentPageNumber={1}
+    defaultPageSize={10}
+    handlePageChange={function noRefCheck() {}}
```

Comment:
Please make sure to go through the React best practices very well. The point of writing those best practices is to ensure that we don't have to again and again provide review comments like these.

https://academy.bigbinary.com/learn-react/pages/prefer-es6-arrow-functions#:~:text=Always%20prefer%20arrow%20functions